src/components/DateRangePicker.js

```diff
@@ -0,0 +1,299 @@
+          height: 20px;
+        }
+      }
+      &.actions {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        & h3 {
+          margin: 0px;
+          color: rgb(43, 42, 47);
+          font-size: 0.9rem;
+        }
+        & svg {
+          &:active {
+            color: rgb(85, 170, 173);
+          }
+          transition: 0.3s;
+          color: rgb(70, 168, 171);
+          width: 30px;
+        }
+      }
+    }
+    & .week {
+      display: flex;
+      & .day {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: calc(100% / 7);
+        padding-top: calc(100% / 7);
+        //height: 20px;
+        position: relative;
+        transition: 0.2s;
+        &:hover {
+          cursor: pointer;
+          &::before {
+            content: ' ';
+            width: 100%;
+            height: 100%;
+            border-radius: 50%;
+            background-color: rgb(202, 201, 201, 0.2);
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            z-index: -1;
+          }
+        }
+        &.selected {
+          &::before {
+            content: ' ';
+            width: 100%;
+            height: 100%;
+            border-radius: 50%;
+            background-color: rgb(34, 182, 184);
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            z-index: -1;
+          }
+          & span {
+            color: white;
+          }
+        }
+        & span {
+          position: absolute;
+          top: 0px;
+          left: 0px;
+          height: 100%;
+          width: 100%;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          color: rgb(82, 82, 82);
+        }
+        &.empty::before {
+          content: ' ';
+          width: 5px;
+          height: 5px;
+          border-radius: 50%;
+          background-color: rgb(228, 228, 228);
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+        }
+      }
+    }
+  }
+`;
+
+const overlay = css`
+  & {
+    position: absolute;
+    height: 100vh;
+    width: 100vw;
+    z-index: 99998;
+    top: 0px;
+    left: 0px;
+    background-color: rgba(40, 40, 40, 0.8);
+    display: none;
+    &.show {
+      display: block;
+    }
+  }
+`;
+
+const daysOfTheWeek = ['S', 'M', 'T', 'W', 'T', 'F', 'S'];
+const monthsNames = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+class Dialog extends Component {
+  constructor() {
+    super();
+    this.state = {
+      days: [],
+      currentMonth: new Date().getMonth(),
+      lastSelection: false,
+      currentYear: 2020,
```

Comment:
new Date().getFullYear() will also work .

Reply:
For the currentYear state?